pipeline-scripts/buildlib.groovy

```diff
@@ -1567,14 +1565,14 @@ def build_plashets(doozerOpts, version, release, dryRun = false) {
         // Populate plashets_built
         plashets_built = readYaml(file: "$working_dir/plashets_built.yaml")
         echo "plashets_built: $plashets_built"
-        if (major > 4 || major == 4 && minor >= 6) { // 4.6+ already fully migrated; no need to create repos on rcm-guest.
+        if (major > 4 || major == 4 && minor >= 6) { // 4.6+ already fully migrated; no need to create repos on ocp-artifacts.
```

Comment:
The comment here is valid. No need to replace the host name.

Reply:
`rcm-guest` will be decommissioned and no longer exist, does it make sense to keep referring to it when it's gone?

Reply:
The comment says "4.6+ fully migrated to ocp-artifacts, so there is no need to create plashet repos on rcm-guest". 
It will not make sense if "rcm-guest" is replaced with "ocp-artifacts" in this sentence.

We can completely remove the comment and the lines after `return`.

Reply:
Fixed in 73de9020807e67ae5cf78f9801dcf9fb9c361e05